src/test/ui/parser/trait-object-trait-parens.rs

```diff
@@ -4,9 +4,7 @@ fn f<T: (Copy) + (?Sized) + (for<'a> Trait<'a>)>() {}
 
 fn main() {
     let _: Box<(Copy) + (?Sized) + (for<'a> Trait<'a>)>;
```

Comment:
shouldn't there also be errors about  `Copy` and `Trait`? Or are they silenced by the undeclared lifetime error? If so, duplicate this file and remove the last `let _` line from the duplicate

Reply:
Yeah, they're silenced by the lifetime error -- the compiler doesn't get far enough. This test is only meant to check parsing, so is the duplication really necessary? (Parsing of brackets aside, this behaviour is already tested elsewhere.)